docs/roadmap/done.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+:::{panels}
+:container: +full-width text-left
+:column: col-lg-6 px-2 py-2
+:card:
+
+**Created on:** June 8, 2022
+
+**Last updated on:** June 8, 2022
+
+:::
+
+## Completed in spring 2022 (Kale 🌱)
+
+Find the full list of grant issues at [Kale](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/5) and [Kale - lessons learned milestones.](https://github.com/Quansight-Labs/jupyter-a11y-mgmt/milestone/8)
+
+- Maintain the [JupyterLab accessibility project board](https://github.com/orgs/Quansight-Labs/projects/5/views/1)
+- Add automated accessibility tests to [jupyter/accessibility](https://github.com/jupyter/accessibility/) for running in JupyterLab (and other configurable Jupyter interfaces) ([JupyterLab issue #9742](https://github.com/jupyterlab/jupyterlab/issues/9742) and [jupyter/accessibility #18](https://github.com/jupyter/accessibility/issues/18))
+- Begin writing custom, manual testing-inspired accessibility tests for JupyterLab
+- Create JupyterLab accessibility statement
```

Comment:
Since we had this discussion, maybe we can reword it to `Created a first iteration of...` or something of the such